.storybook/main.js

```diff
@@ -3,12 +3,10 @@ module.exports = {
 		'./stories/**/*.stories.(js|mdx)'
```

Comment:
This should now be: `'./stories/**/*.stories.@(js|mdx)'`
https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#correct-globs-in-mainjs